config/filter.d/cyrus-imap.conf

```diff
@@ -13,7 +13,7 @@ before = common.conf
 
 _daemon = (?:cyrus/)?(?:imap(d|s)?|pop3(d|s)?)
 
-failregex = ^%(__prefix_line)sbadlogin: \S+ ?\[<HOST>\] \S+ .*?\[?SASL\(-13\): (authentication failure|user not found): .*\]?$
+failregex = ^%(__prefix_line)sbadlogin: (\S+)? ?\[<HOST>\] \S+ .*?\[?SASL\(-13\): (authentication failure|user not found): .*\]?$
```

Comment:
1) In regexp (no matter which re-engine used) is **always** better not to use capturing groups, if you not need to get it. At least it's faster by searching of match and retrieving of really expected groups and fewer resource "intensively".
```python
## non-capture (host only)...
>>> timeit.timeit('m = r.search("test host"); m.group("host")', 
... setup='import re; r = re.compile(r"""^(?:\S*\s)?(?P<host>host)$""")', number=1000000)
0.77633031683956233
## capture (host + no-named one)...
>>> timeit.timeit('m = r.search("test host"); m.group("host")',
... setup='import re; r = re.compile(r"""^(\S*\s)?(?P<host>host)$""")', number=1000000)
0.84158292268296009
```
And it is just a simple test...

2) I was in the mode, where a new expression would be qualitative better, without changing it completely (leading `\[` or not, etc.)